docs/sftp.rst

```diff
@@ -0,0 +1,26 @@
+SFTP Service
+============
+
+In addition to its normal HTTP server, the Girder package also includes a lightweight SFTP
+server that can be used to serve the same underlying data using the
+`SSH File Transfer Protocol <https://en.wikipedia.org/wiki/SSH_File_Transfer_Protocol>`_. This
+server provides a read-only view of the data in a Girder instance, and supports either anonymous
+access (users must pass the username "anonymous" in their SFTP client), or authenticated access
```

Comment:
Do they need to supply any password at all, or blank, in the anonymous case?

Nice docs overall. 💊